src/index.ts

```diff
@@ -6,11 +6,11 @@ import { getNews } from './api/news';
 // import { getNews } from './api/news';
 import { bulkInsertMockData, deleteNote, getNote, getNotes, saveNotes, updateNote } from './routes/crudNotes';
 import getAllUsers from './routes/getAllUsers';
-
+import { NotesController } from './notes/notes_controller';
 
 dotenv.config();
 
-const PORT = process.env.PORT || 3000;
+const PORT = process.env.PORT || 8080;

```

Comment:
any reason, why you changed the default port to 8080? This is common in the Java/Tomcat world.

Reply:
Yes 8080 is common in Java TomEE, but due to some reason my 3000 was showing in use already that's why..
Will make the changes